.gitignore

```diff
@@ -2,3 +2,4 @@
 terraform.tfstate
 *.tfstate*
 terraform.tfvars
+.idea/
```

Comment:
You know, but I just want to mention that you can have global `.gitignore` and include things like `.idea` there. See this - https://gist.github.com/octocat/9257657

Reply:
Yes I know, but I also don't use it for `*.tfstate*`and the likes.